<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="line_PlayListUrl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_LiveMulticastUrl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_PlayListUrl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PlayListUrl</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_LiveMulticastUrl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LiveMulticastUrl</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_MulticastPar01">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MulticastPar01</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_MulticastPar01">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_MulticastPar02">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MulticastPar02</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_MulticastPar02">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>681</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>171</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>now time</string>
   </property>
   <widget class="QLabel" name="lab_Date">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_Time">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>161</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>next time</string>
   </property>
   <widget class="QLabel" name="lab_AfterTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
